app-base/init.sls

```diff
@@ -386,47 +346,28 @@ app.{{ domain }}.{{ instance }}.php-fpm:
       - pkg: php.packages
 {% if pillar['systemd']['apply'] %}
     - onchanges_in:
-      - app.php-fpm.reload
+      - service: app.php-fpm.restart
 {% endif %}
 {% endfor %}
-
-app.{{ domain }}.config:
-  file.managed:
-    - name: {{ platform['user']['home'] }}/config.php
-    - source: salt://app/moodle/config.php.jinja
-    - template: jinja
-    - context:
-      cfg: {{ platform['moodle'] }}
-    - user: {{ platform['user']['name'] }}
-    - group: {{ platform['user']['name'] }}
-    - mode: 0660
 {% endfor %}
 
-{% if pillar['systemd']['apply'] %}
-app.nginx.reload:
-  cmd.run:
-    - name: systemctl reload nginx || systemctl restart nginx
-
-app.nginx.restart:
-  cmd.run:
-    - name: systemctl restart nginx
-
 app.php-fpm.enable:
   service.running:
     - name: php7.0-fpm
     - enable: True
     - require:
       - php.packages
 
-app.php-fpm.reload:
+app.php-fpm.restart:
   cmd.run:
     - name: systemctl reload php7.0-fpm || systemctl restart php7.0-fpm
     - require:
       - php.packages
 
-app.php-fpm.restart:
+app.nginx.reload:
   cmd.run:
-    - name: systemctl restart php7.0-fpm
-    - require:
-      - php.packages
-{% endif %}
+    - name: systemctl reload nginx || systemctl restart nginx
+
+app.nginx.restart:
+  cmd.run:
+    - name: systemctl restart nginx
```

Comment:
None other than that this is what we had in the original role with a reload and a restart thereafter. Happy to remove what was on `master`.